app/controllers/application_controller.rb

```diff
@@ -20,4 +16,10 @@ def authenticate_user
   	redirect_to new_user_sessions_path unless current_user
   end
 
+ def user_not_authorized(exception) #pundit rescue
```

Comment:
Gdzie ta metoda jest wywoływana? Nie widzę jej nigdzie w kodzie. Jeśli odnosisz się do [tego przykładu](https://github.com/elabs/pundit/blob/e8e3a88939e3b7cca720c007603193d251e7ec1b/README.md#rescuing-a-denied-authorization-in-rails) to kluczowa w nim linijka to
```ruby
rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
```
To ona wywołuje ten kod i u Ciebie jej zabrakło. Przeczytaj na spokojnie w dokumentacji co ona dokładnie robi i zastanów się czy na 100% ją chcesz tam mieć?